gbpservice/nfp/service_vendor_agents/haproxy/haproxy_agent.py

```diff
@@ -118,6 +118,11 @@ def setup_ha(self, parent_resource, body):
                                '%s' % parent_resource)
         return methodToCall(body)
 
+    def rsyslog_client_config(self, body):
```

Comment:
All of the visibility features should be extensions to the existing (opensource) service VM agent code.